src/components/Products.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import Product from "./Product"
+import "./Products.css"
+
+function Products({products, addToCart}) {
+    return (
+        <div className = "Products"> 
+           {products.map(product => {
+           return(
+            <Product 
+            key = {product.id}
+            name = {product.name}
+            price = {product.price}
+            img = {product.img}
+            description = {product.description}
+            addToCart = {addToCart}
+           />
```

Comment:
you could add a prop of item={product} so that you can use props.item inside of Product.
This fixes the addToCart functionality.